index.html

```diff
@@ -196,10 +196,10 @@ <h3>
           contribute.
           </li>
           <li>Be aware that displays of affection may complicate professional
-          relationships. For some cultures, overtly friendly disposition
+          relationships. For some people, overtly friendly disposition
           towards another participant involving body contact (e.g.: hugging,
-          touching on the arm or shoulder, or kissing) is uncommon and may be
-          perceived as an invasion of personal space, or as <a>unwelcome advances</a>.
+          touching on the arm or shoulder, or kissing) is uncommon and may be 
+          an invasion of personal space, or perceived as an <a>unwelcome advance</a>.
```

Comment:
I believe hugging and kissing may well **be** an unwelcome advance, not merely perceived as one.